ArduPlane/quadplane.cpp

```diff
@@ -2686,3 +2683,49 @@ float QuadPlane::stopping_distance(void)
     // control the transition distance in a reasonable way
     return plane.ahrs.groundspeed_vector().length_squared() / (2 * transition_decel);
 }
+
+#define LAND_CHECK_ANGLE_ERROR_DEG  30.0f       // maximum angle error to be considered landing
+#define LAND_CHECK_LARGE_ANGLE_CD   1500.0f     // maximum angle target to be considered landing
+#define LAND_CHECK_ACCEL_MOVING     3.0f        // maximum acceleration after subtracting gravity
+
+void QuadPlane::update_throttle_thr_mix(void)
+{
+    // if disarmed or landed prioritise throttle
+    if(!motors->armed()) {
```

Comment:
Copter includes a check of `ap.land_complete` here as well, but I didn't know of an exact corollary in plane...

Reply:
we don't have an equivalent at the moment